src/main/java/uk/gov/hmcts/probate/service/notification/CaveatPersonalisationService.java

```diff
@@ -94,9 +100,31 @@
                 dateFormatterService.formatCaveatExpiryDate(caveatData.getExpiryDate()));
         personalisation.put(PERSONALISATION_WELSH_CAVEAT_EXPIRY_DATE,
                         localDateToWelshStringConverter.convert(caveatData.getExpiryDate()));
+        passDod(caveatData, personalisation);
+        checkIfDobExists(caveatData, personalisation);
 
         return personalisation;
     }
 
+    private void checkIfDobExists(CaveatData caveatData, HashMap<String, String> personalisation){
```

Comment:
perhaps this method should be renamed - getDOBPersonalisation?

Reply:
Done.